db/migrate/20220406011144_add_email_notification_preferences_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailNotificationPreferencesToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :receive_email_notifications, :boolean, default: false
```

Comment:
I'll make those changes, thanks for the review. The validation fails whenever receive_email_notifications and receive_sms_notifications are false. Since they were false by default for all users, that's likely why the seeding error occurred.

Reply:
This seems to be fixed now so resolving!